decentraland/unity-renderer
# feat: support intercom window

## What does this PR change?

Fixes #4490 

1. Modifies the view according to [FIGMA](https://www.figma.com/file/P6mm2kRNslX6GWxDiorEQG/Settings-V2?node-id=1876%3A854&t=Me70LldT05dMqZRp-1) design
2. Adds a Contact support button. In Desktop, it will open a full screen website showing the support in it. In Web, it will open the same popup, but the window will have the size of the popup.
3. Moves `HelpAndSupport` to Addressables following current practices
4. Creates an interface for the view following current practices

## How to test the changes?

1. Open the help and support popup in the settings tab
2. Check that the Discord and FAQs button are working properly
3. Press the Contact Support button. It should open the intercom window. In desktop, its going to be full screen. In web, its gonna have the size of the intercom.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
